gapic/src/main/com/google/gapid/perfetto/models/Tracks.java

```diff
@@ -180,6 +167,20 @@ private Tracks() {
     return data;
   }
 
+  public static Perfetto.Data.Builder enumerateFrame(Perfetto.Data.Builder data) {
+    if (data.getFrame().bufferCount() > 0) {
+      data.tracks.addLabelGroup(null, "sf_events", "Surface Flinger Events",
+          group(state -> new TitlePanel("Surface Flinger Events"), true));
+      String parent = "sf_events";
+      for (FrameInfo.Buffer buffer : data.getFrame().buffers()) {
+        FrameEventsTrack track = FrameEventsTrack.forBuffer(data.qe, buffer);
+        data.tracks.addTrack(parent, track.getId(), buffer.getDisplay(),
+            single(state -> new FrameEventsSummaryPanel(state, buffer, track), true, false));
+      }
+    }
+    return data;
+  }
+
```

Comment:
These tracks should still be in the gpu group, no?

Reply:
Moved them out of that group since the tables are now different as well. They will be just below the GPU for now.